src/librustc_target/spec/aarch64_uwp_windows_msvc.rs

```diff
@@ -0,0 +1,34 @@
+use crate::spec::{LinkerFlavor, Target, TargetResult, PanicStrategy};
+use std::env;
+
+pub fn target() -> TargetResult {
+    let mut base = super::windows_uwp_msvc_base::opts();
+    base.max_atomic_width = Some(64);
+    base.has_elf_tls = true;
+
+    // FIXME: this shouldn't be panic=abort, it should be panic=unwind
+    base.panic_strategy = PanicStrategy::Abort;
+
+    let lib_root_path = env::var("VCToolsInstallDir")
```

Comment:
Note that we already have a good deal of support for this via the `cc::windows_registry` probing that we do in the backend. I think it'd be best to do this there instead of here to not require that the `VCToolsInstallDir` env var is set. Current MSVC targets don't set this env var and I think that with the probing logic we have you should be able to find the root path anyway?

Reply:
Done